cl/search/api_serializers.py

```diff
@@ -414,3 +429,92 @@ class Meta:
             "panel_names",
             "sha1",
         )
+
+
+class RECAPDocumentESResultSerializerBase(DocumentSerializer):
```

Comment:
There's room for improvement in our class naming conventions. For instance, we're adding two parent classes and both use the word "base" inconsistently.

Reply:
Yeah, I renamed it to `BaseRECAPDocumentESResultSerializer`, also updated the docstrings, and renamed `BaseDocketESResultSerializer` to `DocketESResultSerializer` since it's not actually a base class.